crescent/internal/app_command.py

```diff
@@ -114,6 +123,18 @@ def set_id(self, _id: UndefinedOr[Snowflakeish]) -> AppCommand:
         self.id = _id
         return self
 
+    async def create(
+        self,
+        rest: RESTClient,
+        application: SnowflakeishOr[PartialApplication],
+        /,
+        *,
+        guild: UndefinedOr[SnowflakeishOr[PartialGuild]] = UNDEFINED,
+    ) -> PartialCommand:
+        """
+        This method isn't used but someone made it an abstract method in CommandBuilder...
+        """
+
```

Comment:
Are we sure it's not used? Seems like we should make this actually post this command, and just `gather(*(command.create() for command in commands))` under `post_commands`

Reply:
i gotta be honest, i didn't think about that

Reply:
Still unused actually. We use the batch create method so this is never called.